### ISSUE TITLE
ScalaJS project on Scala 2.13.7 looking for non-existent org.scalameta:mtags_2.13.7:0.10.8

### ISSUE THREAD

**Issue by jypma** [scalameta/metals#3302]
**Describe the bug**

I'm trying to compile a ScalaJS 1.7.1 project on Scala 2.13.7 with Metals 0.10.9 (from within Emacs with `lsp-mode`). After import, creating the presentation compiler fails with the following log lines:

```
2021.11.19 12:36:30 ERROR Referring to non-existent class org.scalajs.testing.bridge.Bridge
2021.11.19 12:36:30 ERROR   called from core module module initializers
2021.11.19 12:36:30 ERROR Referring to non-existent method static org.scalajs.testing.bridge.Bridge.start()void
2021.11.19 12:36:30 ERROR   called from core module module initializers
2021.11.19 12:36:30 ERROR JavaScript linking failed with 'There were linking errors'
2021.11.19 12:36:32 INFO  Generated JavaScript file '/home/jan/workspace/scadraw/.bloop/client/client-test.js'
2021.11.19 12:36:32 INFO  no build target: using presentation compiler with only scala-library: 2.13.7
2021.11.19 12:36:34 ERROR failed: Preparing presentation compiler
coursierapi.error.SimpleResolutionError$1: Error downloading org.scalameta:mtags_2.13.7:0.10.8
  not found: /home/jan/.ivy2/local/org.scalameta/mtags_2.13.7/0.10.8/ivys/ivy.xml
  not found: https://repo1.maven.org/maven2/org/scalameta/mtags_2.13.7/0.10.8/mtags_2.13.7-0.10.8.pom
  not found: https://oss.sonatype.org/content/repositories/public/org/scalameta/mtags_2.13.7/0.10.8/mtags_2.13.7-0.10.8.pom
  not found: https://oss.sonatype.org/content/repositories/snapshots/org/scalameta/mtags_2.13.7/0.10.8/mtags_2.13.7-0.10.8.pom
	at coursierapi.error.SimpleResolutionError.of(SimpleResolutionError.java:11)
	at coursierapi.shaded.coursier.internal.api.ApiHelper$.simpleResError(ApiHelper.scala:326)
	at coursierapi.shaded.coursier.internal.api.ApiHelper$.doFetch(ApiHelper.scala:359)
	at coursierapi.shaded.coursier.internal.api.ApiHelper.doFetch(ApiHelper.scala)
	at coursierapi.Fetch.fetchResult(Fetch.java:244)
	at coursierapi.Fetch.fetch(Fetch.java:239)
	at scala.meta.internal.metals.Embedded$.newPresentationCompilerClassLoader(Embedded.scala:271)
	at scala.meta.internal.metals.Embedded.$anonfun$presentationCompiler$2(Embedded.scala:65)
	at scala.meta.internal.metals.StatusBar.trackSlowTask(StatusBar.scala:58)
	at scala.meta.internal.metals.Embedded.$anonfun$presentationCompiler$1(Embedded.scala:65)

```

Incidentally, the only version available online of that library seems to be [0.10.9](https://mvnrepository.com/artifact/org.scalameta/mtags_2.13.7). Version 0.10.8 is indeed nowhere to be found...strange.

The project compiles and runs fine with plain SBT (`sbt`, `project server`, `~reStart`, open `http://localhost:8080/assets/index.html`).

**To Reproduce**

I've shared the basic project on Github, see [`build.sbt`](https://github.com/jypma/scadraw/blob/master/build.sbt).

**Comment by jypma** [comment]
I can add that downgrading to Scala 2.13.6 works around the issue (but the log still shows the same ScalaJS errors, they're probably unrelated).
```
2021.11.19 12:50:56 INFO  time: indexed workspace in 2.48s
2021.11.19 12:50:56 INFO  compiling client (1 scala source)
2021.11.19 12:50:56 INFO  compiling server (2 scala sources)
2021.11.19 12:50:56 ERROR Referring to non-existent class org.scalajs.testing.bridge.Bridge
2021.11.19 12:50:56 ERROR   called from core module module initializers
2021.11.19 12:50:56 ERROR Referring to non-existent method static org.scalajs.testing.bridge.Bridge.start()void
2021.11.19 12:50:56 ERROR   called from core module module initializers
2021.11.19 12:50:56 ERROR JavaScript linking failed with 'There were linking errors'
2021.11.19 12:50:57 INFO  Generated JavaScript file '/home/jan/workspace/scadraw/.bloop/client/client-test.js'
2021.11.19 12:50:58 INFO  no build target: using presentation compiler with only scala-library: 2.13.6
2021.11.19 12:51:06 INFO  time: compiled client in 9.76s
2021.11.19 12:51:06 INFO  time: compiled server in 10s
```
But now code navigation (including into libraries) works fine. Mysterious :-)

**Comment by ckipp01** [comment]
Thanks for the report @jypma, I just tried this out and didn't have any issues with 2.13.7. If you look at the Metals doctor it should show you what version of Bloop is being used. What version does it say?

> I'm trying to compile a ScalaJS 1.7.1 project on Scala 2.13.7 with Metals 0.10.9

You mention 0.10.9 here, but it looks like it's searching for 0.10.8, are you 100% sure you're on 0.10.9? Did you install metals with coursier? If so can you do a `metals -version` to verify?

**Comment by jypma** [comment]
Good point... well, yes and now. I'm running arch, and the [AUR package](https://aur.archlinux.org/packages/metals/) for metals is still on 0.10.8. So I built it myself, figuring that bumping the version number would be enough.

I overlooked that that package apparently [messes with](https://aur.archlinux.org/cgit/aur.git/tree/version-fix.patch?h=metals) the version detection of metals, causing it to make a package with the 0.10.9 sources, thinking they're 0.10.8. I'm rebuilding the package now :) Thanks for the pointers!
